app/serializers/discussion_serializer.rb

```diff
@@ -44,26 +44,26 @@ def self.attributes_from_reader(*attrs)
   has_one :author, serializer: UserSerializer, root: 'users'
   has_one :group, serializer: GroupSerializer, root: 'groups'
   has_one :active_proposal, serializer: MotionSerializer, root: 'proposals'
+  has_one :active_proposal_vote, serializer: VoteSerializer, root: 'votes'
 
-  def author
-    object.author
+  def include_active_proposal_vote?
+    reader.present? && active_proposal.present?
+  end
+
+  def active_proposal_vote
+    active_proposal.votes.find_by(user_id: reader.user_id)
   end
 
   def active_proposal
-    object.current_motion
+    @active_proposal ||= object.current_motion
```

Comment:
I just wanted to cache the `object.current_motion` because we're using it more than once now, and not sure if we're preloading the current motion with the discussion so don't want it to fire off a request twice.

The active proposal's already serialized out with the discussion, this just makes it so that we serialize out the active proposal and your vote for that proposal (if we're serializing out reader info with the discussion)